src/main/java/com/thealgorithms/others/cn/HammingDistance.java

```diff
@@ -0,0 +1,41 @@
+package com.thealgorithms.others.cn;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HammingDistance {
+
+
+    private List<byte[]> extractBinaryData(String senderBits, String receiverBits){
+        List<byte[]> byteArray = new ArrayList<>();
+
+        byteArray.add(senderBits.getBytes());
+        byteArray.add(receiverBits.getBytes());
+
+        return byteArray;
+    }
```

Comment:
Please remove this redundant method

Reply:
Removed the redundant method and moved array inside getHammingDistance() method.